src/core/jsonld.js

```diff
@@ -62,14 +52,13 @@ export function run(conf, doc, cb) {
   }
 
   // normative and informative references
-  jsonld.dependencies = Array.from(conf.normativeReferences).map(ref => addRef(conf, ref));
-  jsonld.citation = Array.from(conf.informativeReferences).map(ref => addRef(conf, ref));
+  const refs = Array.from(conf.normativeReferences)
+    .concat(Array.from(conf.informativeReferences));
+  jsonld.citation = refs.map(ref => addRef(conf, ref));
 
-  var $jsonld = $(
-    "<script type='application/ld+json'>" +
-      JSON.stringify(jsonld) +
-      "</script>"
-  ).appendTo($("head"));
+  var $jsonld = hyperHTML`<script type="application/ld+json"></script>`;
+  $jsonld.appendChild(doc.createTextNode(JSON.stringify(jsonld)));
```

Comment:
`${JSON.stringify(jsonld)}` within script tag should be okay, as that's how hyperHTML works. (It will automatically create a text node)

Reply:
No, it created an error. I found a reference to it here: https://www.danielcrabtree.com/blog/25/gotchas-with-dynamically-adding-script-tags-to-html. I tried your suggestion before resorting to this.

Perhaps, could put the `doc.createTextNode` within the hyperHTML string.

Reply:
Oh, I tried and it breaks only for `<script>`. I didn't know that, thanks!